i18n/zh/docusaurus-plugin-content-docs/current/introduction.md

```diff
@@ -13,7 +13,7 @@ K3s 是轻量级的 Kubernetes。K3s 易于安装，仅需要 Kubernetes 内存
 * Development
 * ARM
 * 嵌入 K8s
-* 不去学习那么多的 Kubernetes 高深知识也能上手使用
+* 无法获得 K8s 集群 PhD 的情况
```

Comment:
This was just changed in https://github.com/k3s-io/docs/pull/118, do you disagree with the translation?